Add jquery-infinitescroll@3.0.0 w/ git auto-update & Update package.json

sufuf3 opened the pull request:
close #11463
Pull request for issue: #11463
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/metafizzy/infinite-scroll
 * Official website (optional, not the repository): https://infinite-scroll.com
 * NPM package url (optional): https://www.npmjs.com/package/infinite-scroll
 * License and its reference: GPL-3.0
 * GitHub / Bitbucket popularity (required): 
   - Count of watchers: 226
   - Count of stars: 4844
   - Count of forks: 1662
 * NPM download stats (optional):
   - Downloads in the last day: 19
   - Downloads in the last week: 117
   - Downloads in the last month: 463

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

sufuf3 commented:
@dakshshah96 @cdnjs/intern3 Please help me review this PR, Thank you.

Note: 
1. The origin GitHub repo: `https://github.com/infinite-scroll/infinite-scroll` always redirect to `https://github.com/metafizzy/infinite-scroll` so I just update https://github.com/cdnjs/cdnjs/tree/master/ajax/libs/jquery-infinitescroll
2. In the first commit, I update `filename`, `version`, `autoupdate` of package.json to fit npm test pass.
3. In the second commit, I update as followings:
  - `homepage`, `repository`, `license`, `author`: as same with upstream.
  - `description`: which show more detail and clear in homepage introduction.
  - `keywords`: Add more keywords from package.json.

sufuf3 commented:
@pvnr0082t @maruilian11 
Please help me review this PR, Thank you.

sufuf3 commented:
@hare1039 @qwe19272375 @pvnr0082t @maruilian11
Please help me review this PR, Thank you.

sufuf3 commented:
@hare1039 @qwe19272375 @pvnr0082t @maruilian11
Please help me review this PR, Thank you.

maruilian11 commented:
@PeterDaveHello Do we also need to mention the commercial license if it has one?

PeterDaveHello commented:
@maruilian11 sure

sufuf3 commented:
@maruilian11 I added `Infinite Scroll Commercial License` this license name into licenses field.
Please help me review this PR, thank you.

PeterDaveHello commented:
@sufuf3 Commercial License should be enough

sufuf3 commented:
@maruilian11 I added `Commercial License` and deleted `Infinite Scroll Commercial License` this license name into licenses field.
Please help me review this PR, thank you.

sufuf3 commented:
@maruilian11 @pvnr0082t 
Please help me review this PR, Thank you.

PeterDaveHello commented on ajax/libs/jquery-infinitescroll/package.json:
```diff
@@ -1,31 +1,38 @@
-  "author": "paulirish"
+  "licenses": [
+    "GPL-3.0",
+    "Commercial License"
```
Use an object to contain its links to the license page please.

sufuf3 commented:
@cdnjs/active-contributor @cdnjs/intern3 I updated the licenses field 
Please help me review this PR, Thank you.

sufuf3 commented:
@extend1994 I changed the order
Please help me review this PR, Thank you.

extend1994 commented:
Hmm I am not sure that should `close` and `cc` exchange the location. It seems weird that use https://github.com/cdnjs/cdnjs/pull/11551/commits/13cafe5aafe5b24a96710b64d7b67e36baff2b99 to close the corresponding issue.

sufuf3 commented:
@extend1994 I modified the commit message.
Please help me review this PR, thanks.

sufuf3 commented:
@cdnjs/active-contributor @cdnjs/intern3 
Please help me review this PR, Thank you.
